package/MDAnalysis/analysis/align.py

```diff
@@ -756,6 +756,7 @@ def __init__(self, mobile, reference, select='all', filename=None,
           for the remainder of the Python session. If ``mobile.trajectory`` is
           already a :class:`MemoryReader` then it is *always* treated as if
           ``in_memory`` had been set to ``True``.
+        - Writer ``**kwargs`` will be passed to the constructed `mda.Writer`.
```

Comment:
use `writer_kwargs` dictionary instead to differentiate them from normal `kwargs`. Some writers cannot accept e.g. `verbose`.

Reply:
@ljwoods2 Please leave review comments unresolved; reviewers may need to verify that the comments have indeed been addressed. :)